src/karan/T1_firstNonReaptingChar.java

```diff
@@ -0,0 +1,31 @@
+package karan;
+
+public class T1_firstNonReaptingChar {
+	
+	void display(String input){
+		for(int i=0;i<input.length();i++) {
+			
+			boolean flag = true;
+			
+			for(int j=0; j<input.length();j++){
```

Comment:
here you can use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop